examples/amp-story/invalid-link-tests.html

```diff
@@ -16,26 +16,29 @@
         background: white;
       }
       #link1 {
-        position: absolute;
-        top: 300px;
-        right: 320px;
+        position: relative;
+        top: 50%;
+        left: -10%;
```

Comment:
Changing the reference point to left makes sense but we will still get movement if using percentage since it's based on viewport width. 

Link1 is moving around:
![Nov-24-2020 15-36-58](https://user-images.githubusercontent.com/3860311/100148777-34f85380-2e6b-11eb-98a1-b3c4c69c6f10.gif)

We might want `left: -40px` if we want the position to be locked there.
-10% works for the example though so this is just a suggestion :)

Reply:
I realized that the safe space is based on percentage here so maybe percentage does make sense :) 
I think either way works for the example though.